core/src/test/java/org/elasticsearch/search/searchafter/SearchAfterIT.java

```diff
@@ -102,7 +103,7 @@ public void testsShouldFail() throws Exception {
                     .addSort("field1", SortOrder.ASC)
                     .setQuery(matchAllQuery())
                     .searchAfter(new Object[]{1})
-                    .get();
+                    .execute().actionGet();
```

Comment:
I changed it to make it consistent with the other calls in this test but I agree that `.get()` is less typing ;)